nano/node/bootstrap.cpp

```diff
@@ -2326,15 +2437,15 @@ class request_response_visitor : public nano::message_visitor
 			}
 		});
 	}
-	void publish (nano::publish const &) override
+	void publish (nano::publish const & message_a) override
```

Comment:
Since we don't use message_a this will give unused variable warnings, omitting the name suppresses the warning.

Reply:
Sure